CHANGELOG.md

```diff
@@ -2,6 +2,86 @@
+- Improve the orbit eye to always maintain an up-axis [#5193](https://github.com/rerun-io/rerun/pull/5193)
+- Focus on current bounding-box when resetting camera-eye on a 3D space view (double click it) [#5209](https://github.com/rerun-io/rerun/pull/5209)
+- Add STL mesh support [#5244](https://github.com/rerun-io/rerun/pull/5244)
+- Add first person 3D eye-camera [#5249](https://github.com/rerun-io/rerun/pull/5249)
+
+#### 🚀 Performance Improvements
+- More robust handling of maximum texture size for non-color data, slight perf improvements for large point clouds [#5229](https://github.com/rerun-io/rerun/pull/5229)
+- Cached transforms & disconnected spaces for faster scenes with many transforms [#5221](https://github.com/rerun-io/rerun/pull/5221)
+- Optimized cpu time for 3D point clouds (once again!) [#5273](https://github.com/rerun-io/rerun/pull/5273)
+- Only compute store/caching stats when the memory panel is opened [#5274](https://github.com/rerun-io/rerun/pull/5274)
+
+#### 🧑‍🏫 Examples
+- Add Gesture Recognition example [#5241](https://github.com/rerun-io/rerun/pull/5241) (thanks [@andreasnaoum](https://github.com/andreasnaoum)!)
+
+#### 📚 Docs
+- Fix broken link in the installing-viewer documentation [#5236](https://github.com/rerun-io/rerun/pull/5236) (thanks [@BirgerMoell](https://github.com/BirgerMoell)!)
+
+#### 🖼 UI Improvements
+- Context Menu 1: Basic scaffolding and simple actions [#5163](https://github.com/rerun-io/rerun/pull/5163)
+- Context menu 2: add support for multiple selection [#5205](https://github.com/rerun-io/rerun/pull/5205)
+- Context menu 3: add "Move to new container" context menu action [#5210](https://github.com/rerun-io/rerun/pull/5210)
+- Context menu 4: add "Clone space view" action [#5265](https://github.com/rerun-io/rerun/pull/5265)
+- Clickable path parts in selection-panel [#5220](https://github.com/rerun-io/rerun/pull/5220)
+- Don't show the blueprint section when selecting recordings [#5245](https://github.com/rerun-io/rerun/pull/5245)
+- Use the same icon for recordings everywhere [#5246](https://github.com/rerun-io/rerun/pull/5246)
+
+#### 🎨 Renderer Improvements
+- Lift point cloud size limitations [#5192](https://github.com/rerun-io/rerun/pull/5192)
+- Lift line vertex/strip count limitations [#5207](https://github.com/rerun-io/rerun/pull/5207)
+- Fix banding artifacts of 3D space view's skybox [#5279](https://github.com/rerun-io/rerun/pull/5279)
+
+#### 🗣 Refactors
+- Context menu 5: refactor into multiple files [#5289](https://github.com/rerun-io/rerun/pull/5289)
+
+#### 📦 Dependencies
+- Bump maturin to 1.14.0 [#5197](https://github.com/rerun-io/rerun/pull/5197)
+- Update `tungstenite` to remove RUSTSEC warning [#5200](https://github.com/rerun-io/rerun/pull/5200)
+- Lock the web-sys version to 0.3.67 [#5211](https://github.com/rerun-io/rerun/pull/5211)
+- Increase the max WebSocket frame limit for the native client [#5282](https://github.com/rerun-io/rerun/pull/5282)
```

Comment:
Arguably this should go under "Performance Improvements" since it is a user-facing improvement in the size of the things they can stream